core/base-service/coalesce-badge.spec.js

```diff
@@ -0,0 +1,257 @@
+'use strict'
+
+const { expect } = require('chai')
+const { getShieldsIcon } = require('../../lib/logos')
+const coalesceBadge = require('./coalesce-badge')
+
+describe('coalesceBadge', function() {
+  describe('Label', function() {
+    it('uses the default label', function() {
+      expect(coalesceBadge({}, {}, { label: 'heyo' }).text).to.deep.equal([
+        'heyo',
+        'n/a',
+      ])
+    })
+
+    // This behavior isn't great and we might want to remove it.
```

Comment:
What's the thinking on removing it? I actually just noticed this not too long ago when I forgot to set a default badge label and noticed the category being used in my local env. If we removed this, would there be an error thrown (programmer error) or what would the label show as alternatively?

Reply:
Throwing an error is a good idea.

Reply:
I think @chris48s was the first person who articulate this was a bad/unexpected use of the category.